opm/material/fluidsystems/Spe5ParameterCache.hpp

```diff
@@ -175,6 +175,25 @@ class Spe5ParameterCache
         };
     }
 
+    /*!
+     * \brief TODO
+     *
+     * \param phaseIdx The fluid phase of interest
+     * \param compIdx The component phase of interest
+     * \param compJIdx Additional component index
+     */
+    Scalar aCache(unsigned phaseIdx, unsigned compIdx, unsigned compJIdx)
+    {
+        switch (phaseIdx)
+        {
+        case oilPhaseIdx: return oilPhaseParams_.getaCache(compIdx,compJIdx);
+        case gasPhaseIdx: return gasPhaseParams_.getaCache(compIdx,compJIdx);
+        default:
+            throw std::logic_error("The aCache() parameter is only defined for "
+                                   "oil phase");
```

Comment:
Should this diagnostic message mention the gas phase too?  There does at least seem to be code to handle the gas phase here.

Reply:
yes, will add the gas phase